src/main/java/seedu/address/logic/commands/EditMeetingCommand.java

```diff
@@ -102,8 +105,6 @@ private static Meeting createEditedMeeting(Meeting meetingToEdit, EditMeetingDes
         }
         Meeting meeting = new Meeting(meetingToEdit.getClient(), updatedDescription, updatedDate,
                 updatedStartTime, updatedEndTime);
-        meeting.getClient().removeMeeting(meetingToEdit);
```

Comment:
these are side effects that shouldnt be in here. these causes huge issues as it doesn't obey atomicity, so any transaction that fails will have side effects, instead of being reverted. this is huge bc with this, some of the erroneous edits made by the transaction is still applied.